tests/acceptance/bootstrap/CliContext.php

```diff
@@ -501,12 +533,14 @@ public function theCLIResponseShouldContainTheseEntries(string $shouldOrNot, Tab
 	 */
 	public function getJSONDecodedCliMessage(ResponseInterface $response): array {
 		$responseBody = $this->featureContext->getJsonDecodedResponse($response);
-
 		// $responseBody["message"] contains a message info with the array of output json of the upload sessions command
 		// Example Output: "INFO memory is not limited, skipping package=github.com/KimMachineGun/automemlimit/memlimit [{<output-json>}]"
 		// So, only extracting the array of output json from the message
-		\preg_match('/(\[.*\])/', $responseBody["message"], $matches);
-		return \json_decode($matches[1], null, 512, JSON_THROW_ON_ERROR);
+		\preg_match('/(\[.*?\])/', $responseBody["message"], $matches);
+        if (!isset($matches[1])) {
+            return [];
+        }
```

Comment:
```suggestion
        if (!isset($matches[1])) {
            return [];
        }
```

Return empty array when no matches found